architecture/core_concepts/builds_and_image_streams.adoc

```diff
@@ -175,29 +175,157 @@ endif::[]
-. Image repositories from external registries
+* An external registry, for example `registry.access.redhat.com` or `hub.docker.com`
+* Other image streams in the {product-title} cluster
 
-ifdef::openshift-origin,openshift-online,openshift-enterprise,openshift-dedicated[]
-Image streams can be used to automatically perform an action when new images are
-created. Builds and deployments can watch an image stream to receive
-notifications when new images are added and react by performing a build or
-deployment, respectively. For a
-curated set of image streams, see the
+When you define an object that references an image stream tag (such as a or build configuration or deployment configuration), you point to an image stream tag, not the Docker repository. When you build or deploy your application, {product-title} queries the Docker repository using the image stream tag to locate the associated ID of the image and uses that 
+exact image.  
+
+The image stream metadata is stored in the etcd instance along with other cluster information.
+
+The following image stream contains two tags: `34` which points to a Python v3.4 image and `35` which points to a Python v3.5 image:
+
+----
+oc describe is python
+Name:			python
+Namespace:		imagestream
+Created:		25 hours ago
+Labels:			app=python
+Annotations:		openshift.io/generated-by=OpenShiftWebConsole
+			openshift.io/image.dockerRepositoryCheck=2017-10-03T19:48:00Z
+Docker Pull Spec:	docker-registry.default.svc:5000/imagestream/python
+Image Lookup:		local=false
+Unique Images:		2
+Tags:			2
+
+34
+  tagged from centos/python-34-centos7
+
+  * centos/python-34-centos7@sha256:28178e2352d31f240de1af1370be855db33ae9782de737bb005247d8791a54d0
+      14 seconds ago
+
+35
+  tagged from centos/python-35-centos7
+
+  * centos/python-35-centos7@sha256:2efb79ca3ac9c9145a63675fb0c09220ab3b8d4005d35e0644417ee552548b10
+      7 seconds ago
+----
+
+Using image streams has several significant benefits:
+
+* You can tag, rollback a tag, and quickly deal with images, without having to re-push using the command line.
+
+* You can trigger Builds and Deployments when a new image is pushed to the registry. Also,
+{product-title} also has generic triggers for other resources (such as Kubernetes objects).
+
+* You can xref:../../dev_guide/managing_images.adoc#importing-tag-and-image-metadata[mark a tag for periodic re-import].
+If the source image has changed, that change is picked up and reflected in the image stream, which trigger the Build and/or Deployment flow, depending upon the Build or Deployment configuration.
+
+* You can share images using fine-grained access control and quickly distribute images across your teams.
+
+* If the source image changes, the image stream tag will still point to a known-good version of the image, ensuring that your application will not break unexpectedly.
+
+* You can configure security around who can view and use the images through permissions on the image stream objects. 
```

Comment:
Do you know if we cover this in the docs? Not sure what I am looking for in how to set image permissions.

Reply:
probably not explicitly for imagestreams, they are controlled in the same way access is controlled to any resource: https://docs.openshift.org/latest/admin_guide/manage_authorization_policy.html